### DISCUSSION TITLE
Hot keys for putting on/off diving suit

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#13959] [Ideas and suggestions]
After the last update, the diving suits became too slow.
But walking without a suit is a way of delayed suicide.
As a result, barotrauma has become a game of “take off the diving suit and then put it on”
There is an idea! Make a mod that helps to take on/off the diving suit.
We just need 2 hotkeys.
The first key is to take off the suit and grab it in your hands, and if the suit already is in your hands, then put it on.
The second key should take off the suit and drop it to the ground.
As an example for creating the hotkeys there are 2 mods
https://steamcommunity.com/sharedfiles/filedetails/?id=2958095512
https://steamcommunity.com/sharedfiles/filedetails/?id=2768813474
I could figure it out myself, but I'll have to start from scratch.
What do you think of the idea? Looks interesting?
Can anybody help?

**Comment by T1MM5H** [comment]
I'd implement this so that "attacking" with a held suit has the same result as Hotkey'ing a clothing item in your Hotbar. I'm just not sure if held items support that sort of thing, or if they are a different class of item.
